code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -338,3 +344,13 @@
 	// Add verb for re-opening the interview panel, fixing chat and re-init the verbs for the stat panel
 	add_verb(src, /mob/dead/new_player/proc/open_interview)
 	add_verb(client, /client/verb/fix_tgui_panel)
+
+///Resets the Lobby Menu HUD, recreating and reassigning it to the new player
+/mob/dead/new_player/proc/reset_menu_hud()
+	set name = "Reset Lobby Menu HUD"
+	set category = "OOC"
+	var/mob/dead/new_player/new_player = usr
+	if(new_player?.client)
+		qdel(new_player.hud_used)
+		create_mob_hud()
+		hud_used.show_hud(hud_used.hud_version)
```

Comment:
do you mind placing a cooldown on this (see `COOLDOWN_START` and friends)? huds involve a lot of subdatums (particularly plane masters), and I don't want to create a lag vector.

Reply:
https://github.com/tgstation/tgstation/pull/76443/commits/9735e710e838bb6be51edc9c03dbe0e9688a8a48